book/src/nix/editing-book.md

```diff
@@ -0,0 +1,37 @@
+# Editing this Book
+_So meta_
+
+---
+
+The editing process of this book has also been nixified.
+
+## Serving a hot-reloading version of this book
+
+To serve a hot-reloading version of this book, simply run:
+
+```bash
+nix run ".#serve-book"
+```
+
+And open [http://localhost:3000](http://localhost:3000/) in your browser.
+
+Then, edit the sources in `book/src/`, and see them update live in your browser!
+
+
+## Checking your spelling
+
+After adding new content, you probably want to check your spelling. To do this, run the following:
+
+```bash
+nix build ".#cspell-check"'
```

Comment:
It will be renamed to `spell-check` here https://github.com/ComposableFi/composable/pull/1711/files#diff-206b9ce276ab5971a2489d75eb1b12999d4bf3843b7988cbe8d687cfde61dea0R895

Reply:
ah thanks, i will already rename it then